vars/main.yml

```
@@ -12,7 +12,15 @@ __mssql_required_facts_subsets: "{{ ['!all', '!min'] +
   __mssql_required_facts }}"
 __mssql_server_packages: mssql-server
 __mssql_server_selinux_packages: mssql-server-selinux
-__mssql_client_packages: [mssql-tools, unixODBC-devel]
+__sqlcmd_ver: "{{
```

```suggestion
__sqlcmd_ver: "{{ mssql_tools_versions | sort | last }}"
```
This should work if `length == 1` or `length > 1` - it seems a little unnecessary but 1) the underlying python code will probably be faster without the if condition 2) this is simpler

One thing - is there a way to make `sort` do a numeric comparison?  It doesn't really matter until version `100` which will be sorted before `99`, or if they change the way they do versions

---

Right, one line is better.
I hope that they end on version 18... or make all future versions backward-compatible and move the current mssql-tools to an exceptional `mssql-tools17`. So that other packages can be simply updated. We can act accordingly.